rc/pdfpcrc

```diff
@@ -1,129 +1,123 @@
 #### Keybindings
 
-# Forward movement
-bind Next      next
-bind Return    next
-bind Right     next
-bind space     next
-bind l         lastOverlay
-bind Down      nextOverlay
-bind S+Next    next10
-bind S+Return  next10
-bind S+Right   next10
-bind S+space   next10
-bind S+Down    next10
-bind k         nextUnseen
-
-# Back movement
-bind Left      prev
-bind Prior     prev
-bind L         firstOverlay
-bind Up        prevOverlay
-bind S+Left    prev10
-bind S+Prior   prev10
-bind S+Up      prev10
-bind j         prevSeen
+# Standard forward movement
+bind Next               next
+bind Return             next
+bind Right              next
+bind space              next
+bind Down               next
+
+# Standard backward movement
+bind Prior              prev
+bind Left               prev
+bind Up                 prev
+bind BackSpace          prev
+
+# Overlay paging
+bind S+Next             nextOverlay
+bind S+Prior            prevOverlay
+
+# Intra-slide navigation
+bind S+Up               lastOverlay
+bind S+Down             firstOverlay
+
+# Fast forward/backward (10 pages)
+bind S+Right            next10
+bind S+Left             prev10
 
 # "Jumps"
-bind Home      gotoFirst
-bind End       gotoLast
-bind g         goto
-bind Tab       overview
-bind BackSpace histBack
+bind Home               gotoFirst
+bind End                gotoLast
+bind g                  goto
+
+# Bookmarking [the current action names are misleading]
```

Comment:
Since we are breaking it anyways, why not changing the names?

Reply:
Sure, if you agree this is indeed a bookmarking operation (albeit limited to just one bookmark...) I personally use this functionality to reload a presentation if I make changes during rehearsal. But if in future we implement real reload, I see no need for these at all.

Reply:
Then we could delete it, but if we make things more consistent now, then this can be changed.

Reply:
Actually, it goes deeper, as the "bookmarked" page number is save in the .pdfpc file under the `[last_saved_slide]` name. If we change it there, a minor backward incompatibility will be created. Shall we go for it?

Reply:
BTW, on load, if the given slide is an overlay, the last one will be shown. Not sure if this is intended.

Reply:
OK, I renamed these actions without touching the function names and code in general.